packages/main/test/samples/ColorPalette.sample.html

```diff
@@ -7,7 +7,7 @@ <h2 class="control-header">ColorPalette</h2>
 <div class="control-tag">&lt;ui5-color-palette&gt;</div>
 
 <section>
-	<h3>Basic ColorPalette</h3>
+	<h3>Color Palette</h3>
 	<div class="snippet">
 		<ui5-color-palette>
 			<ui5-color-palette-item value="darkblue"></ui5-color-palette-item>
```

Comment:
We use value here, but we no longer have a value property

Reply:
The value is on the `ui5-color-palette-item`, there we have `value`